AdRulesIMAWidevineSampleApp/src/main/java/com/brightcove/player/samples/imawidevine/adrules/MainActivity.java

```diff
@@ -60,7 +60,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // Create the catalog object which will start and play the video.
         Catalog catalog = new Catalog("FqicLlYykdimMML7pj65Gi8IHl8EVReWMJh6rLDcTjTMqdb5ay_xFA..");
```

Comment:
You'd need to do a reverse lookup on the video ID to get the catalog id, which im pretty sure you can't do. Is there another ID to identify your videocloud account other than the catalog id? Wouldn't this 'videocloud' id also be a token and thus messy?